scripts/mitochondria_m2_wdl/MitochondriaPipelineSlim.wdl

```diff
@@ -0,0 +1,657 @@
+version 1.0
```

Comment:
Could you please add some example input jsons (possibly for both hg38 and hg19)?

Reply:
Added.  The following commands need to be run by Sam Novod before these JSON files will work (hopefully he will do this asap): 
   gsutil cp /seq/references/Homo_sapiens_assembly38/v0/chrM/chrM.hg38.interval_list gs://gcp-public-data--broad-references/hg38/v0/chrM/chrM.hg38.interval_list
   gsutil cp /seq/references/Homo_sapiens_assembly19/v1/MT/MT.hg19.interval_list  gs://gcp-public-data--broad-references/hg19/v0/MT.hg19.interval_list